packages/react-components/react-card/src/components/Card/useCardStyles.ts

```diff
@@ -60,15 +61,6 @@ const useStyles = makeStyles({
     ...shorthands.borderRadius(tokens.borderRadiusLarge),
   },
 
-  interactiveNoOutline: {
-    ':hover::after': {
-      ...shorthands.borderColor(tokens.colorTransparentStrokeInteractive),
-    },
-    ':active::after': {
-      ...shorthands.borderColor(tokens.colorTransparentStrokeInteractive),
-    },
-  },
```

Comment:
Merged into each separate appearance style to simplify assignment logic later on.